pypom/page.py

```diff
@@ -107,13 +107,31 @@ def open(self):
         raise UsageError('Set a base URL or URL_TEMPLATE to open this page.')
 
     def wait_for_page_to_load(self):
-        """Wait for the page to load.
+        """
+        Wait for the page to load.
 
         By default the driver will try to wait for any page loads to be
-        complete, however it's not uncommon for it to return early. To address
-        this you can override :py:func:`wait_for_page_to_load` and implement an
-        explicit wait for a condition that evaluates to ``True`` when the page
-        has finished loading.
+        complete, however it's not uncommon for it to return early.
+
+        :py:func:`wait_for_page_to_load` is a template method. Do not override it.
+        Instead, use :py:func:`_before_wait_for_page_load` or :py:func:`_after_wait_for_page_load`
+        for pre- and post- actions.
+
+        Override :py:func:`_is_loaded` and implement an explicit wait for a
```

Comment:
Replace `_is_loaded` with `_page_loaded`, and note that this is a property that returns True. There's no need for the user to write an explicit wait themselves.